app/javascript/controllers/simple_tree_controller.js

```diff
@@ -8,15 +8,17 @@ export default class extends Controller {
 
   connect () {
     setTimeout(() => {
-      let activeElem = this.element.querySelector('.tree-link.active');
-      if (activeElem) {
-        activeElem.scrollIntoView({ block: 'center' });
-        window.scrollTo({top: 0,});
-        if (this.autoClickValue) {
-          activeElem.click();
+      if(window.location.href.includes('p=classes')){
```

Comment:
too specific, need to be in a value or deduced somehow

Reply:
This code is used only in the concept details jump to (specific to it). I'll extract it to a function with a good name to make the code more clear

Reply:
this controller is used in all the places where there is a tree,

Reply:
that part of the code scrolling the tree container to the selected node.

Reply:
whatever which type of tree, properties, schemes, ....

Reply:
Yes, but this code is related only for the tree view of the concepts cause it's the only one that has a specific jump to with a page reloading, and for the others the jump to filter the results directly.
This is already deployed to StagePortal test it to understand me.

Reply:
You are correct could be used for all the tabs with tree view when we load the page.
To be safe I will specify all of the pages with tree view: p=concepts, p=instances, p=properties, p=schemes, p=collections